ui/src/app/pages/analysis/runtime-panel.spec.tsx

```diff
@@ -159,6 +159,55 @@ describe('RuntimePanel', () => {
     expect(runtimeApiStub.runtime.gceConfig.machineType).toEqual('n1-standard-4');
   });
 
+  it('should create runtime with preset values instead of getRuntime values if configurationType is GeneralAnalysis', async() => {
+    const gceConfig = defaultGceConfig();
+    gceConfig.machineType = 'n1-standard-16';
```

Comment:
This test case is great. I would put a comment in here somewhere to explain more specifically the scenario this is covering.